src/components/EditProfile.vue

```diff
@@ -9,8 +9,9 @@
         <div class="modal-body">
             <label class="form-label">
               Profile Picture
-              <input type="file" class="form-control" @change="update($event.target.files)" accept="image/*">
+              <input type="file" class="form-control" @change="update" accept="image/*">
```

Comment:
It only allows for one, but I did see the issue with the thumbnails not being reset so it looks like multiple images are uploading. I fixed that and will push the changes up soon.